lib/repl.js

```diff
@@ -689,8 +690,31 @@ function intFilter(item) {
   return /^[A-Za-z_$]/.test(item);
 }
 
+const DEFAULT_PROPERTIES = {
```

Comment:
To be clear: I'm fine with it either way, it just looks a bit weird to me in this specific case, especially the uppercased properties, not the object name itself.